src/components/footer/z-footer/index.tsx

```diff
@@ -92,30 +107,70 @@ export class ZFooter {
 
   renderFooterBottom(): HTMLElement {
     return (
-      <section class="bottom">
-        <div class="item logo">
-          {this.renderZLogo()}
-          {this.renderCopyright()}
-          {this.renderCertification()}
-        </div>
-        <div class="item">
-          {this.renderAddress()}
-          {this.renderSocial()}
+      <section id="bottom">
+        <div class={{ "limited-width": !!this.contentMaxWidth }} style={this.contentMaxWidth ? { "--mw": `${this.contentMaxWidth}px` } : {}}>
+          <div class="item logo">
+            {this.renderZLogo()}
+            {this.renderCopyright()}
+            {this.renderCertification()}
+          </div>
+          <div class="item">
+            {this.renderAddress()}
+            {this.renderSocial()}
+          </div>
+          {this.renderBottomLinks()}
         </div>
-        {this.renderBottomLinks()}
       </section>
     );
   }
 
   renderFooterTop(): HTMLElement {
     return (
-      <section class="top">
-        <slot />
-        {this.renderFooterTopJsonData()}
+      <section id="top">
+        <div class={{ "limited-width": !!this.contentMaxWidth }} style={this.contentMaxWidth ? { "--mw": `${this.contentMaxWidth}px` } : {}}>
```

Comment:
To avoid having to set up the --mw variable and give the limited-width class to all 3 sub-components you could use the same logic that has been used in the topbar component in which a content-container div is used to wrap all the sub-components.

Reply:
Unluckily the 3 sub-components have different background colors (at least 2 of them), so we cannot use a unique container to set a common left/right margin